source/settings/core.rst

```diff
@@ -2052,6 +2052,10 @@ See :ref:`settings` for list of all setting groups.
 
       SiS with immediate byte-by-byte comparison during saving.
 
+      SiS is deprecated and writing of SiS files is disabled.
+      Reading is supported,
+      missing SiS attachments are replaced with files filled with spaces.
```

Comment:
A bit clearer: `Reading is supported for now. Any missing SIS attachments ...`

Also, I see the previous text already used SiS but that is just wrong. It's not "Single in Storage" but "Single Instance Storage". So now would be a good time to change also the existing SiS to SIS.